### PR TITLE
Make gh-pages intro header clear

### PR DESCRIPTION
**Pull request by hsol**
# Make gh-pages intro header clear
## Issue
No issue

## Details
Deletes:
- Delete position absolute on header content
- Delete min height on header

Changes:
- Change bottom padding on header content
- Change some media query width value

Add:
- Set font size 11px on nav item when width less than 860px
- Set header content padding top 100px when width less than 860px

### CODE REVIEW

**Comment by netil** [comment]
Thanks @hsol for the contribution!

Could you add/update padding-top values for header content as below?

```css
header .header-content {
    ...
    padding: 10px 15px 50px 15px;


@media(max-width:860px) {
	...
    header .header-content {
        padding-top: 100px;
    }
```

**Comment by hsol** [comment]
I don't know if this is what you wanted(4931cdc5438c503bbc7968f38ab6d20125a60e9c)

File: css/styles.css [hunk 1]
```diff
@@ -259,7 +259,7 @@ header {
 header .header-content {
     position: relative;
     width: 100%;
-    padding: 100px 15px;
+    padding: 100px 15px 50px 15px;
```
Comment (netil) [inline css/styles.css]: Set padding-top 10px. It's having unnecessary blank space on the top.
```css
padding: 10px 15px 50px 15px;
```

Comment (hsol) [reply css/styles.css]: Was fix here. f6b8186463e9a6567397b72d003f903adb52f7d1

**Pull request merged**